clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -1176,10 +1176,11 @@ public static String toHtml() {
         for (ApiKeys key : ApiKeys.values()) {
             // Key
             b.append("<h5>");
+            b.append("<a name=\"The_Messages_" + key.name + "\">");
```

Comment:
What's the deal with the `name` attribute instead of `id`? From what I can gather about html versions, `name` isn't actually valid in HTML, even HTML5, and `id` is the correct attribute to use.